src/catalogue/items/CatalogueItemsDetailsPanel.component.tsx

```diff
@@ -268,60 +274,79 @@ function CatalogueItemsDetailsPanel(props: CatalogueItemsDetailsPanelProps) {
         <TabPanel value={tabValue} index={2}>
           <Grid item container spacing={0}>
             <Grid item xs={12}>
-              <ListItem
-                style={{
-                  justifyContent: 'flex-start',
-                  display: 'flex',
-                  flexDirection: 'row',
-                  alignItems: 'center',
-                  padding: 0,
-                }}
-              >
-                <ListItemText
-                  primary={'Manufacturer Name'}
-                  secondary={catalogueItemIdData.manufacturer.name}
-                />
-              </ListItem>
+              <Grid item xs={12}>
+                <Typography color="text.primary">Manufacturer Name</Typography>
+                <Typography color="text.secondary">
+                  {manufacturerData?.name}
+                </Typography>
+              </Grid>
+            </Grid>
+            <Grid item xs={12}>
+              {manufacturerData?.url ? (
+                <ListItem
+                  style={{
+                    justifyContent: 'flex-start',
+                    display: 'flex',
+                    flexDirection: 'row',
+                    alignItems: 'center',
+                    padding: 0,
+                  }}
+                >
+                  <ListItemText
+                    primary={'Manufacturer URL'}
+                    secondary={
+                      <MuiLink
+                        underline="hover"
+                        target="_blank"
+                        href={manufacturerData?.url}
+                      >
+                        {manufacturerData?.url ?? 'None'}
+                      </MuiLink>
+                    }
+                  />
+                </ListItem>
```

Comment:
Could you also apply the ListItem change to Typographies to these and the others in the file (assuming they have no parent list) - they are all the same as you have already done.